Export/Scripts/pantheons.lua

```diff
@@ -0,0 +1,62 @@
+--
+-- export The Pantheon data
+--
+
+local function zip(a, b)
```

Comment:
`zip` is a universally useful function to have, so it would be better if it lived in `Common.lua`. That way, other parts of the program can also take advantage of it.